src/main/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheck.java

```diff
@@ -68,7 +68,7 @@
  * <li>
  * Property {@code header} - Specify the required header specified inline.
  * Individual header lines must be separated by the string {@code "\n"}
- * (even on platforms with a different line separator), see examples below.
+ * (even on platforms with a different line separator).
```

Comment:
This is not expected. Why is this changed?

If it is because Javadoc has no examples, this is still copied to xdoc which does.